Tests/test_PAML_codeml.py

```diff
@@ -472,6 +473,24 @@ def testParsePairwise(self):
             pairwise = results["pairwise"]
             self.assertEqual(len(pairwise), 5, version_msg)
 
+    def testParseSitesParamsForPairwise(self):
+        """Verify that pairwise site estimates are indeed parsed. Fixes #483"""
+        res_dir = os.path.join(self.results_dir, "codeml", "pairwise")
+        for results_file in os.listdir(res_dir):
+            version = results_file.split('-')[1].split('.')[0]
+            version_msg = "Improper parsing for version %s" \
+                        % version.replace('_', '.')
+            results_path = os.path.join(res_dir, results_file)
+            results = codeml.read(results_path)
+            for seq2 in results['pairwise'].values():
+                for data in seq2.values():
+                    for param in ("t", "S", "N", "omega", "dN", "dS", "lnL"):
+                        self.assertTrue(param in data, version_msg +
+                                        ": '%s' not in parsed parameters" % param)
+                        self.assertTrue(isinstance(data[param], numbers.Number))
```

Comment:
Thanks. Please open an issue on taking advantage of the numbers module in Biopython if you think it would be useful.